app/assets/javascripts/modules/track-click.js

```diff
@@ -10,14 +10,24 @@ window.GOVUK.Modules = window.GOVUK.Modules || {};
       var options = {},
           category = element.attr('data-track-category'),
           action = element.attr('data-track-action'),
-          label = element.attr('data-track-label');
+          label = element.attr('data-track-label'),
+          dimension = element.attr('data-track-dimension'),
+          customDimension = element.data('track-custom-dimension');
 
       if (label) {
         options.label = label;
       }
 
       function trackClick() {
-        if (GOVUK.analytics && GOVUK.analytics.trackEvent) {
+        if (!GOVUK.analytics) {
+          return;
+        }
+
+        if (customDimension && dimension && GOVUK.analytics.setDimension) {
+          GOVUK.analytics.setDimension(customDimension, dimension);
```

Comment:
As a side-note, I'm not sure we do track downloads? For example the PDF on the [Computing Curriculum](https://www.gov.uk/government/publications/national-curriculum-in-england-computing-programmes-of-study) page doesn't have any tracking markup, and I don't see any events fired off to GA in my Network tab. Unless it's a specific type of download?